examples/reference/templates/FastGridTemplate.ipynb

```diff
@@ -150,6 +154,9 @@
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "7fa204e84b1ce6697eb3048458244c17ccc160da5eb825bd435eb4b5077a9bf7"
+  },
```

Comment:
Any idea where this came from?

Reply:
Nope

Reply:
Do you agree that the `pn.bind` api probably is the "best api" to communicate to users?

Reply:
Trying out new slogans for the template

![image](https://user-images.githubusercontent.com/42288570/127129801-d69603fd-6ec6-42f0-b370-37c392ca2ca1.png)

:-)

Reply:
Yes, @jbednar and I agree that `pn.bind` is most powerful.

Reply:
I've removed the hash.